tutorials/animation/introduction.rst

```diff
@@ -146,15 +146,15 @@ toolbar:
    Convenience buttons
 
 These switches and buttons allow you to add keyframes for the selected
-node's location, rotation, and scale respectively.
+node's location, rotation, and scale. Since we are only animating the sprite's 
+position, make sure that only the location switch is selected. (The selected 
+switches are blue.)
```

Comment:
```suggestion
node's location, rotation, and scale. Since we are only animating the sprite's 
position, make sure that only the location switch is selected. The selected 
switches are blue.
```